features/all-positions/LiquidationActionDialog.tsx

```diff
@@ -0,0 +1,488 @@
+                </Status>
+                <Status>
+                  <Label>Liquidator: </Label>
+                  <a
+                    href={getEtherscanUrl(liquidatedPosition.liquidator)}
+                    target="_blank"
+                  >
+                    {prettyAddress(liquidatedPosition.liquidator)}
+                  </a>
+                </Status>
+
+                <Status>
+                  <Label>Disputer: </Label>
+                  {liquidatedPosition.disputer && (
+                    <a
+                      href={getEtherscanUrl(liquidatedPosition.disputer)}
+                      target="_blank"
+                    >
+                      {prettyAddress(liquidatedPosition.disputer)}
+                    </a>
+                  )}
+                  {!liquidatedPosition.disputer && (
+                    <span>None(undisputed)</span>
+                  )}
+                </Status>
+                <Status>
+                  <Label>Liquidation Transaction: </Label>
+                  <a
+                    href={getEtherscanUrl(
+                      liquidatedPosition.liquidationReceipt
+                    )}
+                    target="_blank"
+                  >
+                    {prettyAddress(liquidatedPosition.liquidationReceipt)}
+                  </a>
+                </Status>
+                <Status>
+                  <Label>Liquidation ID: </Label>
+                  {liquidatedPosition.liquidationId}
+                </Status>
+                <Status>
+                  <Label>Tokens liquidated: </Label>
+                  {prettyBalance(
+                    Number(liquidatedPosition.tokensLiquidated)
+                  )}{" "}
+                  {tokenSymbol}
+                </Status>
+                <Status>
+                  <Label>Locked Collateral: </Label>
+                  {prettyBalance(
+                    Number(liquidatedPosition.lockedCollateral)
+                  )}{" "}
+                  {collSymbol}
+                </Status>
+                <Status>
+                  <Label>
+                    {invertDisputablePrice ? `Minimum` : `Maximum`} Disputable
+                    Price:{" "}
+                  </Label>
+
+                  {prettyBalance(
+                    getDisputablePrice(liquidatedPosition.maxDisputablePrice)
+                  )}
+                </Status>
+                <hr />
+                <Box pt={2}>
+                  <Typography>
+                    <strong>Dispute this liquidation</strong>
+                    <br></br>
+                  </Typography>
+                  {translateLiquidationStatus(
+                    Number(liquidatedPosition.liquidationTimestamp),
+                    liquidatedPosition.status
+                  ).status !== "Liquidation Pending" && (
+                    <span>
+                      <Typography>
+                        This liquidation can't be disputed as it is no longer in
```

Comment:
This is a bit awkward to read. "This liquidation can't be disputed because the liveness period to file disputes has passed"

Reply:
done.